src/converter/modules/ExlClient.js

```diff
@@ -93,6 +95,53 @@ export default class ExlClient {
     throw new Error(`Playlist with id: ${id} not found`);
   }
 
+  /**
+   * Get non-English labels from EXL Config populated endpoints
+   * @param {string} endpoint
+   * @param {string} id
+   * @param {string} lang
+   * @returns {string}
+   */
+  async getLabelFromEndpoint(endpoint, id, lang = 'en') {
+    if (LABELS_FROM_ENDPOINTS[endpoint] === undefined) {
+      LABELS_FROM_ENDPOINTS[endpoint] = {};
+      LABELS_FROM_ENDPOINTS[endpoint][lang] = {};
+    }
+
+    if (LABELS_FROM_ENDPOINTS.endpoint?.lang === undefined) {
+      let next = `api/${endpoint}?lang=${lang}&page_size=2000`;
+
+      do {
+        /* eslint-disable-next-line no-await-in-loop */
+        const response = await this.doFetch(next);
+
+        if (response.error) {
+          console.error(response.error);
+        } else {
+          const raw = response?.data;
+
+          if (raw === undefined || raw.length <= 0) {
+            console.error(`${endpoint} request returned no labels for ${lang}`);
+          }
+
+          raw.forEach((item) => {
+            const enLabel =
+              item.Name_en === undefined ? item?.Name : item?.Name_en;
+            LABELS_FROM_ENDPOINTS[endpoint][lang][enLabel] = item.Name;
```

Comment:
If we assume that we dont use this for `en` can't we remove this?

Reply:
Yep, good call